packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -77,44 +77,79 @@ export function useSliderThumb(
   stateRef.current = state;
   let reverseX = direction === 'rtl';
   let currentPosition = useRef<number>(null);
+
+  let pageSize = 1;
```

Comment:
Should default page size be `1` or something like `(getThumbMaxValue(index) - getThumbMinValue(index)) / 10`, so that `PageUp`/`PageDown` behaves like a native `input[type=range]`?

Reply:
Also, should `pageSize` be a public API distinct from `step`? So that things like ColorSlider can increment or decrement RGB values by `0x10` using `PageUp`/`PageDown` while `step` remains `0x1`.

Reply:
> Also, should `pageSize` be a public API distinct from `step`? So that things like ColorSlider can increment or decrement RGB values by `0x10` using `PageUp`/`PageDown` while `step` remains `0x1`.

I think it can be, not sure if we want to open that in this PR just to minimize changes we make. Would it simplify some of our code elsewhere? That would be a reason to do it.



> Should default page size be `1` or something like `(getThumbMaxValue(index) - getThumbMinValue(index)) / 10`, so that `PageUp`/`PageDown` behaves like a native `input[type=range]`?

Interesting, I didn't pick up that that was what native was doing. I think that'd be ok.

Reply:
`(getThumbMaxValue(index) - getThumbMinValue(index)) / 10` might be weird for RangeSlider where the `min` and `max` for each thumb value is constrained. I think this is yet another reason for a public API that distinguishes `pageSize` from `step`.